superset/config.py

```diff
@@ -1095,6 +1095,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 
 # SQLalchemy link doc reference
 SQLALCHEMY_DOCS_URL = "https://docs.sqlalchemy.org/en/13/core/engines.html"
+SQLALCHEMY_DISPLAY_TEXT = "SQLAlchemy docs"
```

Comment:
small nit, but we could also make this a dictionary to keep the values together.